c2cgeoportal/forms.py

```diff
@@ -299,6 +307,7 @@ def render_item(self, item, depth):
             "name": self.name + ("-second" if final_item.id in self._rendered_id else ""),
             "value": final_item.id,
             "add": ' checked="checked"' if self.is_checked(item, final_item) else "",
+            "type": prefixs.get(final_item.item_type, ""),
```

Comment:
Perhaps you can add a comma (",") at the end of line 311 (under the new "type" value). Just to be constant.